example_test.go

```diff
@@ -41,3 +41,20 @@ func ExampleNewAmount() {
 	// 0 BTC
 	// invalid bitcoin amount
 }
+
+func ExampleAmount_unitConversions() {
+	amount := btcutil.Amount(44433322211100)
+
+	fmt.Println("Satoshi to kBTC:", amount.Format(btcutil.AmountKiloBTC))
+	fmt.Println("Satoshi to BTC:", amount.String())
```

Comment:
Not that it matters a lot, but no need to call `.String()` here. `btcutil.Amount` is a `Stringer`, so `fmt.Println` will automaticaly invoke it:
```Go
fmt.Println("Satoshi to BTC:", amount)
```